controllers/machinehealthcheck_controller_test.go

```diff
@@ -175,6 +177,54 @@ func TestMachineHealthCheck_Reconcile(t *testing.T) {
 		))
 	})
 
+	t.Run("it ignores Machines not matching the label selector", func(t *testing.T) {
+		g := NewWithT(t)
+		ctx := context.TODO()
```

Comment:
```suggestion
```

There should be a global context already defined, if there isn't you can definitely ignore this